brainzutils/mail.py

```diff
@@ -22,29 +23,35 @@ def send_mail(subject, text, recipients, attachments=None,
         from_name: Name of the sender.
         from_addr: Email address of the sender.
     """
-    if attachments is None:
-        attachments = []
+
     if from_addr is None:
         from_addr = 'noreply@' + current_app.config['MAIL_FROM_DOMAIN']
-
+              
     if current_app.config['TESTING']:  # Not sending any emails during the testing process
         return
 
     if not recipients:
         return
 
-    message = MIMEMultipart('mixed')
+    message =MIMEMultipart()
+
+    if boundary is not None:
+        originalboundary = "===============2220963697271485568=="
```

Comment:
I don't like this, in general please avoid using magic constants in the code. Each constant should be named.

you should just do 


```
if boundary is not None:
    message = MIMEMultipart(boundary=boundary)
else:
    # set message to default value here
```

Reply:
Quite dumb of me. Thankss :)